### ISSUE TITLE
Is it possible to use different lora models in different regions?

### ISSUE THREAD

**Issue by KimigaiiWuyi** [comfyanonymous/ComfyUI#374]
I don't know if this is even possible...
I noticed that in the last Ksampler node, the same model is always connected to that node.
In the last step, it seems to be contaminated by the Lora model for the whole area composition?

**Comment by Season02** [comment]
I have the exactly same question

**Comment by pxjhll** [comment]
I have the exactly same question +1

**Comment by ShyRed** [comment]
Same question. Just started messing aroundwith regions in 1111 and using `Composable Lora` plugin you can use different LoRA per region.

Would like to know if that is possible with ComyUI, too. A small example would be awesome.

**Comment by daizhuo** [comment]
I tried multi laten composition with different lora, but had no luck. Could anyone answer this question? Thank you!

**Comment by MaxLanc** [comment]
You kinda "can", not like the extension in auto1111 tho, but by pure cope and hope using the nodes.
TLDR you connect the model with all the loras to the KSampler but with like 0.5 strength on each lora and each region prompt has connected the model clip and their own lora clip as well.

You are not limiting the lora influence, just letting them interfere enough to have their intended effects and then on the high res pass you kinda can fix the artifacts and weirdness.

**Comment by daizhuo** [comment]
> You kinda "can", not like the extension in auto1111 tho, but by pure cope and hope using the nodes. TLDR you connect the model with all the loras to the KSampler but with like 0.5 strength on each lora and each region prompt has connected the model clip and their own lora clip as well.
> 
> You are not limiting the lora influence, just letting them interfere enough to have their intended effects and then on the high res pass you kinda can fix the artifacts and weirdness.

Great 👍 Thank you so much!
Could you attach one simple sample how to do that? That would be awesome!

**Comment by MaxLanc** [comment]
Sure thing, fair warning the results won't be as crisp and clear as they should but so far this is the only way without actually messing with the code lol

This is one such result
![ComfyUI_00064_](https://github.com/comfyanonymous/ComfyUI/assets/95586248/aa1c30dd-80e5-465b-938f-084a86a5753c)

And this is the copium machine I noded in comfyUI, is pretty normal but it took me some seethe mald and cope to achieve lol
![imagen](https://github.com/comfyanonymous/ComfyUI/assets/95586248/0f971399-4a6f-43ec-960c-c5f399c861f9)
![imagen](https://github.com/comfyanonymous/ComfyUI/assets/95586248/bc6b044e-b904-443d-a746-ad7efa9a415e)
![imagen](https://github.com/comfyanonymous/ComfyUI/assets/95586248/27466057-493b-47b9-a164-fd298446eb62)

**Comment by MaxLanc** [comment]
> Great 👍 Thank you so much! Could you attach one simple sample how to do that? That would be awesome!

Forgot the quote, check answer up lol
